modules/parent-join/src/main/java/org/elasticsearch/join/aggregations/ParentToChildrenAggregator.java

```diff
@@ -55,7 +57,7 @@
     private final String parentType;
```

Comment:
Also this field can now be removed as it is no longer used. 
(it can also be removed from the `ChildrenAggregatorFactory`)